plugins/kubernetes/app/models/kubernetes/deploy_yaml.rb

```diff
@@ -127,6 +128,10 @@ def set_replica_target
       template[:spec][:replicas] = @doc.replica_target if template[:kind] == 'Deployment'
     end
 
+    def set_name
+      template[:metadata][:name] = @doc.kubernetes_role.resource_name
```

Comment:
deployment.metadata.name / daemonset.metadata.name
<http://deployment.metadata.name/> / job.metadata.name

On Wed, Jun 29, 2016 at 9:50 AM, rata <notifications@github.com> wrote:

> In plugins/kubernetes/app/models/kubernetes/deploy_yaml.rb
> <https://github.com/zendesk/samson/pull/1110#discussion_r68984223>:
>
> > @@ -127,6 +128,10 @@ def set_replica_target
> >        template[:spec][:replicas] = @doc.replica_target if template[:kind] == 'Deployment'
> >      end
> >
> > +    def set_name
> > +      template[:metadata][:name] = @doc.kubernetes_role.resource_name
>
> What is metadata.name? deployment.metadata.name? (or job.metadata.name or
> daemonset...)?
>
> I mean, in terms of kubectl explain, what do you refer to? Just to be
> sure I understand
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1110/files/69e70907f8826375ada24e525c7a1777046671ee#r68984223>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ8VJQGPipVyg-pQXGMfCT6rNa2Gsks5qQqJYgaJpZM4JAoD7>
> .
>